Docs: Slightly more details regarding relations on the item object (#19381)
directus/directus: docs/reference/items.md

::: tip Relational Data

> The items being returned here don't contain related or nested data. To retrieve related data see

Reviewer: ```suggestion
The items being returned here don't contain related or nested data. To retrieve related data, see
```

> The items being returned here don't contain related or nested data. To retrieve related data, see
> [Relational Data](/reference/introduction#relational-data) and [Field Parameters](/reference/query#fields).

Reviewer 2: Assuming the following simple example collection setup:
```
a
  id
  b - Many to One to 'b'

b
  id
```
If an item from `a` is requested, by default it contains info about the relation, but no nested data about this relation:
```jsonc
{
  "id": 1,
  "b": 1, // primary key of b
}
```

Hence my suggestion:

```suggestion
By default, the item object doesn't contain nested relational data. To retrieve nested data, see [Relational Data](/reference/introduction#relational-data) and [Field Parameters](/reference/query#fields).
```

::: tip Relational Data

The item object contains only physical relationships, which means that no alias relationships are returned here. To
retrieve _all_ relationships see [Relational Data](/reference/introduction#relational-data) and

> [Field Parameters](/reference/query#fields).

Reviewer 2: As far as I know, we don't need the term "physical relationships" anywhere else.
So instead of the proposed text, I would probably just borrow the one from https://docs.directus.io/reference/introduction.html#relational-data:
> Directus only retrieves the fields in your items that explicitly have been requested. Relational data can be retrieved nested by using [the fields parameter](https://docs.directus.io/reference/query.html#fields) in REST, or regular nested queries in GraphQL.

Author: that sound's way better but it doesn't seem to be correct actually. when I make a GET request on  `/items/:collection/:pk` I receive all fields without explicitly requesting anyone 🤔

Reviewer 2: Wondering if we should update the mentioned sentence as well 😄 

> Directus only retrieves the **relational** fields in your items that explicitly have been requested. Relational data can be retrieved nested by using [the fields parameter](https://docs.directus.io/reference/query.html#fields) in REST, or regular nested queries in GraphQL.
